util/env_posix.cc

```diff
@@ -216,30 +229,15 @@ class PosixEnv : public Env {
         }
       }
       close(fd);
```

Comment:
this is a comment for the existing code: I don't understand why we close the file descriptor here, is it a bug?